packages/components/src/public/QueryModel/withQueryModels.tsx

```diff
@@ -123,6 +123,11 @@ const paramsEqual = (oldParams, newParams): boolean => {
     return false;
 };
 
+/**
+ * For in depth documentation and examples see components/docs/QueryModel.md
```

Comment:
Should we add this note to the other top level items that are doc'ed in `QueryModel.md`? i.e. GridPanel, DetailPanel, QueryModel, etc.